knowledge-base/actions/datadog/labeler/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
```

Comment:
Please update input to input: repo-token, view [here](https://github.com/DataDog/labeler/blob/1a043751b2faccb4f9078d542928e4c0d3f0e2e5/action.yml#L5)